src/TestHeaders.am

```diff
@@ -5,21 +5,31 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-## Test ./*.h files in a directory using test-suite/testheaders.sh
+# A space-separated list of header files to be skipped by
+# testHeaders during "make check"
+EXCLUDE_FROM_HDR_TESTING =
 
-## TODO: include in src/Common.am when all testHeaders users are the same.
 
-TESTS += testHeaders
-
-## .h dependency test script
-## aborts build process on errors; XXX: even with "make -k"
-testHeaders: $(srcdir)/*.h
-	$(SHELL) $(top_srcdir)/test-suite/testheaders.sh "$(CXXCOMPILE)" $^ || exit 1
+# to test additional header files to those tested by default,
```

Comment:
I do not insist on this change that attempts to polish wording. FWIW, "additional X to Y" phrasing sounds awkward to me.

```suggestion
# to test header files not tested by default,
```

or

```suggestion
# to test header files in addition to those tested by default,
```